test/test.js

```diff
@@ -0,0 +1,141 @@
+const mocha = require('mocha')
+const { describe, it } = mocha
+
+const assert = require('assert')
+
+const windows = require('..')('1.8')
```

Comment:
Have you tested this with other versions as well?

Reply:
no

to my knowledge, the click modes implemented here have not changed much since 1.8